### PR TITLE
Updates to proctoring for learners

### PR DESCRIPTION
**Pull request by srpearce**
## [DOC-3721](https://openedx.atlassian.net/browse/DOC-3721)

Updates to proctoring. Mentions that links to rules have been added in Studio and the LMS, and includes reorganization and word-count reduction for learner guide.

Also updates docs to say that the score is automatically set to 0 when proctoring result is "Unsatisfactory", but this info has been commented out. It will be re-added in a later pull request.

### Date Needed (optional)

9 Aug 2017

### Reviewers

Possible roles follow. The PR submitter checks the boxes after each reviewer finishes and gives :+1:. 

- [ ] Doc team review (copy edit): @edx/doc

### Testing

- [ ] Ran ./run_tests.sh without warnings or errors

### HTML Version (optional)

- [ ] http://draft-proctoring-for-learners-aug-2017.readthedocs.io/en/latest/completing_assignments/proctored_exams/index.html

### Sandbox (optional)

- [ ] Point to or build a sandbox for the software change and add a link here

### Post-review

- [ ] Add a comment with the description of this change or link this PR to the next release notes task.
- [ ] Squash commits

### CODE REVIEW

**Comment by catong** [comment]
@srpearce Looks good! Tagging @dhixonedx so that the Support team is aware of changes. 

Also, a reminder to update links from the LMS and from the Help Center to Learner Guide pages, as the restructuring might have broken these links.

**Comment by catong** [comment]
👍

**Comment by srpearce** [comment]
@thallada and @katymywy, FYI, this is the PR for automatically setting the grade to 0 for an "Unsatisfactory" score. Because the "set grade to 0" is a one-sentence change for learners and course authors, and this PR also reorganizes the proctoring docs for learners (a re-org for course authors is in the works), you don't need to review it. I can release these docs as soon as the feature is live.

**Comment by thallada** [comment]
@srpearce I'm confused about the difference between these two:

* http://edx.readthedocs.io/projects/edx-guide-for-students/en/latest/
* http://edx-guide-for-students.readthedocs.io/en/latest/

It seems like they both resolve to the same place. Are we trying to move from one URL to the other? Should I be using only one of those, and if so which one?

**Comment by srpearce** [comment]
@thallada That's a very good question. The correct URL is http://edx.readthedocs.io/projects/edx-guide-for-students/en/latest/, though you're right—they do look like they go to the same place. Where did you find the second one?

**Comment by srpearce** [comment]
@thallada Oh, NVM - I see my earlier comment. In any case, RTD defines that as the "short URL".

**Comment by thallada** [comment]
@srpearce I see. I'll just use
http://edx.readthedocs.io/projects/edx-guide-for-students/en/latest/ in the
proctoring configuration URLs so they are all consistent.

**Comment by srpearce** [comment]
@thallada Perfect. Thanks!

File: en_us/course_authors/source/course_features/proctored_exams/CA_online_proctoring_rules_students.rst [hunk 1]
```diff
@@ -4,36 +4,27 @@
...
-
-As with other ID-verified assessments and exams, learners will be asked to
-establish their identity during the proctored exam process by supplying a photo
-ID with a name that matches the full name they use in their edx.org account.
-
-.. note:: Learners who require additional time or any specific accommodations
-   or exemptions to the exam policies in order to complete a proctored exam,
-   are asked to contact their course teams or edX Support. For information
-   about handling special requests relating to proctored exams, see
-   :ref:`Respond to Learner Concerns about Proctored Exams` and
```
Comment (srpearce) [inline en_us/course_authors/source/course_features/proctored_exams/CA_online_proctoring_rules_students.rst:32]: Replaced the above with the below.

File: en_us/course_authors/source/course_features/proctored_exams/proctored_results.rst [hunk 2]
```diff
@@ -22,10 +22,12 @@ flagged.
 
 Two results are possible for proctoring session reviews.
 
-*  **Satisfactory** - the learner has passed the proctoring review.
+*  **Satisfactory** - The learner has passed the proctoring review.
 
-*  **Unsatisfactory** - Some suspicious activity has been observed. The
-   learner has not passed the proctoring review.
```
Comment (srpearce) [inline en_us/course_authors/source/course_features/proctored_exams/proctored_results.rst:28]: Added information to the following.

File: en_us/course_authors/source/course_features/proctored_exams/proctored_troubleshooting.rst [hunk 3]
```diff
@@ -16,8 +16,10 @@ Navigating to Another Part of the Course During an Exam
 While they are taking a proctored exam, learners can navigate (using the same
 browser session) to other parts of your course.
 
-.. note:: It is a violation of edX's Online Proctoring Rules for learners to
-   navigate to websites other than edX.org during the proctored exam.
```
Comment (srpearce) [inline en_us/course_authors/source/course_features/proctored_exams/proctored_troubleshooting.rst:20]: Added an xref below.

File: en_us/students/source/completing_assignments/proctored_exams/SFD_proctoring_overview.rst [hunk 4]
```diff
@@ -0,0 +1,43 @@
+.. _SFD Proctored Exams Overview:
```
Comment (srpearce) [inline en_us/students/source/completing_assignments/proctored_exams/SFD_proctoring_overview.rst:1]: Revised this file.

**Review by srpearce — COMMENTED** [review]
@catong  I've made some revisions that could use a once-over. I've added comments inline to point out the revisions.

**Comment by catong** [comment]
@srpearce Thanks for the tag. It's a bit difficult to follow all the changes in the github diff, but what I can see looks fine. A couple of caveats re restructuring - we should keep any anchor labels the same even if the visible heading changes, to minimize broken links. New sections obviously need new anchors, but when we move sections and change titles, we should leave the existing anchors as they are.

**Pull request merged**